src/components/Preview.css

```diff
@@ -60,17 +60,17 @@
   color: #fff;
 }
 
-.Component-header {
+.Preview-header {
   display: flex;
   justify-content: center;
   align-items: center;
 }
 
-.Component-header h2 {
+.Preview-header h2 {
```

Comment:
If all items within .Preview-header are white in color, just set it as such. For example:

```
.Preview-header {
    color: white;
}
```

That said, why is this header the only one with specified color?